app/assets/javascripts/discourse/app/models/reviewable.js

```diff
@@ -12,6 +13,25 @@ export const REJECTED = 2;
 export const IGNORED = 3;
 export const DELETED = 4;
 
+const DEFAULT_COMPONENT = "user-menu/default-reviewable-item";
+function defaultItemComponents() {
+  return {
+    ReviewableFlaggedPost: "user-menu/reviewable-flagged-post-item",
+    ReviewableQueuedPost: "user-menu/reviewable-queued-post-item",
+    ReviewableUser: "user-menu/reviewable-user-item",
+  };
+}
+
+let _itemComponents = defaultItemComponents();
+
+export class MiniReviewable extends RestModel {
```

Comment:
I wonder if there is a better class name we can use here to reflect that this model is meant for the notifications menu.

Reply:
It'll also be better if we can move the logic for this model into its dedicated file.